src/components/Image/types.ts

```diff
@@ -0,0 +1,38 @@
+import { TPAComponentProps } from '../../types';
+
+export interface ImageProps extends TPAComponentProps {
+  /** The source could be any absolute full URL or a relative URI of a media platform item */
+  src?: string;
+  /** The intrinsic width of the image in pixels */
+  width?: number;
+  /** The intrinsic height of the image in pixels */
+  height?: number;
+  /** The alternative text description of the image. Allowing better SEO when the image has meaning of content */
+  alt?: string;
+  /** A callback to be called when the image is loaded */
+  onLoad?: React.EventHandler<React.SyntheticEvent>;
+  /** A callback to be called if error occurs while loading */
+  onError?: React.EventHandler<React.SyntheticEvent>;
+  /** Specifies how the image is resized to fit its container */
+  resize?: 'contain' | 'cover';
+  // Specifies the proportional relationship between width and height
+  aspectRatio?: 'square' | 'cinema' | 'landscape' | number;
```

Comment:
I think in this case what I would do is just accept a number, and pass the AspectRatios as "preset".
That way, consumer could use:
```jsx
aspectRatio={AspectRatios.square}
```

Alternatively we can write the AspectRatios as an enum , and have:
```tsx
const enum AspectRatios {
  square = 1,
  cinema = 16 / 9,
  landscape = 4 / 3
}
...
aspectRatio?: AspectRatios | number
```

Reply:
Same dilemma like `resize`. If you think it still better I'll change